cpp/test/computed_fields_test.cc

```diff
@@ -35,6 +35,12 @@ TEST(ComputedFieldsTests, AccessFields) {
 
   r.vector_of_vectors_field = {{1, 2, 3}, {4, 5, 6}};
   EXPECT_EQ(r.AccessVectorOfVectorsField(), r.vector_of_vectors_field[1][2]);
+
+  r.map_field = {{"hello", "world"}, {"world", "bye"}};
+  EXPECT_EQ(r.AccessMap(), r.map_field);
+  EXPECT_EQ(r.AccessMapEntry(), "world");
+  EXPECT_EQ(r.AccessMapEntryWithComputedField(), "world");
```

Comment:
I am wondering if we should have a test that confirms that we throw if the key is not there. Right now we rely on https://en.cppreference.com/w/cpp/container/unordered_map/at to do that, so we are pretty sure that is the case, but if we later on switch the underlying implementation or we use this set of tests as a template later to write tests in another language, then we might miss it. Note that the `[]` operator of unordered_map creates the entry if it doesn't exist (https://en.cppreference.com/w/cpp/container/unordered_map/operator_at).